roles/bootstrap-os/tasks/bootstrap-debian.yml

```diff
@@ -44,28 +44,28 @@
     - not skip_http_proxy_on_os_packages
 
 - name: Check Network Name Resolution configuration
-  raw: grep '^DNSSEC=allow-downgrade' /etc/systemd/resolved.conf
+  raw: grep '^DNSSEC=yes' /etc/systemd/resolved.conf
   register: need_dnssec_allow_downgrade
   failed_when: false
   changed_when: false
   # This command should always run, even in check mode
   check_mode: false
   when:
-    - '''UBUNTU_CODENAME=bionic'' in os_release.stdout_lines'
+    - '"UBUNTU_CODENAME=bionic" in os_release.stdout_lines'
```

Comment:
Is this part necessary?
The change reason is not written on this PR message.

The above double ' comes from https://github.com/kubernetes-sigs/kubespray/pull/7269 and the PR changes the other parts also.

Reply:
I've used 2 `'` instead of `"` to match what was used in the other cases in #7269
This is purely cosmetic indeed

Reply:
No, I'll revert. I do find it harder to read when using the same quote character for both the YAML String and the nested Jinja2/Python string unnecessarily. The rest of the code base uses `'"` or `"'` whereas `'''` is only used in bootstrap-os which seems to be introduced by #7269. 

E.g.
```
$ git grep -l \'\'\' -- '*.yml'
roles/bootstrap-os/tasks/bootstrap-centos.yml
roles/bootstrap-os/tasks/main.yml

$ git grep -l "\(\"'\)\|\('\"\)"  -- '*.yml' | wc -l
42
```

Most style guides for languages that can use single or double discourage it, e.g.

* https://prettier.io/docs/en/rationale.html#strings
* https://www.python.org/dev/peps/pep-0008/#string-quotes
* https://google.github.io/styleguide/jsguide.html#features-string-literals

Though there aren't any specific ones that come to mind for Ansible, the https://github.com/ansible/ansible pretty much entirely uses `"'` or `'"` for their tests and documentation.

Reply:
If you feel like it, submit a PR changing all those to `'" ... "'`